agr_literature_service/api/models/person_model.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict
+from sqlalchemy import Column, Integer, String, ARRAY, Index, UniqueConstraint
+from sqlalchemy.orm import relationship
+from agr_literature_service.api.database.base import Base
+from agr_literature_service.api.database.versioning import enable_versioning
+from agr_literature_service.api.models.audited_model import AuditedModel
+
+enable_versioning()
+
+
+class PersonModel(Base, AuditedModel):
+    __tablename__ = "person"
+    __versioned__: Dict = {}
+
+    person_id = Column(Integer, primary_key=True, autoincrement=True)
+    display_name = Column(String(), nullable=False)
```

Comment:
Same here, add indexes on `display_name`

Reply:
display_name is indexed